tensorflow_lite_support/ios/BUILD

```diff
@@ -79,3 +79,14 @@ tflite_ios_framework(
         "//tensorflow_lite_support/c/task/vision:image_classifier",
     ],
 )
+
+objc_library(
+    name = "TFLCommon",
+    hdrs = [
+        "Sources/TFLCommon.h",
```

Comment:
What is the common code hierarchy of obj-c? I'm not familiar with obj-c, but wondering what the code style convention is. 

The current pattern is:
ios/
 - task/
   -  core/
      - Sources/ 
   - text/
     - api/
     - nlclassifier/
       - Sources/
       - Tests/
     - qa/
       - Sources/
       - Tests/
   - vison/
- Sources/ 
- text/
   - tokenizers/
     - Sources/
     - Tests/
 - utils/
 
Should we make Sources/ a root directory under ios/? For example:
ios/
  - Sources/
    - task/
      - core/
      - text/
      - vision/
    - TFLCommon.h
  - Tests

Reply:
There is no standard specified in ObjC AFAIK. A lot of popular frameworks don't use the `sources` folder at all. They just group them in a hierarchy meaningfully. But most of the iOS frameworks from Google maintain a similar hierarchy. I just followed whatever was followed for the text library.  If you look at [tensorflow lite objc](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/objc), they have two folders `sources` and `apis`. `Sources` are all the `.mm` files and internal headers. `Apis` are public headers. Task library omits api folder and puts everything in `Sources`. We can follow the structure you mentioned if it doesn't interfere with any Google standards. Won't matter make a difference from the iOS POV.  @khanhlvg Do you have more info about this?

Reply:
I think Lu's suggestion on the folder structure is more scalable. The only concern is that would it require significant changes on the build script for the current Text library?

```
ios/
  Sources/
    task/
       core/
       text/
       vision/
    TFLCommon.h
Tests
```

@lu-wang-g Do you know if Prianka can make the folder restructure or do we need to do it internally?

Reply:
Let's loop in Youngsoek and see if it makes sense to reorganize the code structure and if it will cause backward incompatibility issue. This PR can be merged first.

Reply:
I can proceed with this hierarchy. There is also an apis folder in [ios/task/text](https://github.com/tensorflow/tflite-support/tree/master/tensorflow_lite_support/ios/task/text/apis) with umbrella header and module map. Where would these files go in the new structure? apis don't go inside sources, right? So will there be another apis folder like

Reply:
But we will have to create a hierarchy for task in apis as well in that case right?
For eg:,
ios/
 apis/
    task/
      text/
        TFLTask.h
        framework.modulemap
      vision/
        TFLVision.h
        framework.modulemap
 Sources/
    task/
      text/
      vision/

Reply:
I think so, there's another folder, apis/, to host the header files.

@priankakariatyml how about we start to follow the new structure for vision tasks. If you have extra bandwidth, you can help us to refactor the text tasks' code. Otherwise, I'll handle it when I have chance.  

Seems like the code is better to be organized according to each iOS package, i.e. for text / vision / audio, respectively. So the new structure could be something like this:
ios/
  Sources/   <= General source code for tflite_support. Doesn't have to be used in Task library.
     TFLCommon.h
  apis/ 
  task/
     core/
        Sources/  <= should this be "sources/"?  It's less common to capitalize directory name in Google's code 
        apis/
     text/
        Sources/ 
        apis/
     vision/
        Sources/ 
        apis/

What do you think? Please feel free to let me know. Thanks!

Reply:
This looks good to me as well since there is an apis folder too. I can reorganise text and update it in the next PR.  So if I understand correctly, all public headers go into respective `api` folders. `.m` and `internal headers` go into `sources`. Am I right?